jest.config.int.js

```diff
@@ -1,4 +0,0 @@
-const config = require('./jest.config');
```

Comment:
Why did we remove that?
are the tests still running?

Reply:
It was an unused file breaking with the newest jest/tsjest

Reply:
thanks resolving